src/main/java/hudson/plugins/logparser/LogParserConsts.java

```diff
@@ -25,12 +26,12 @@ public class LogParserConsts {
 
     public static String getHtmlOpeningTags() {
         final String hudsonRoot = Jenkins.get().getRootUrl();
+        final String pluginResourceUrl = String.format("%s/plugin/log-parser/", Jenkins.RESOURCE_PATH).substring(1);
+
         return "<!DOCTYPE html>\n" + "<html>\n" + "\t<head>\n"
                 + "\t\t<title>log-parser plugin page</title>\n"
-                + "\t\t<link type=\"text/css\" rel=\"stylesheet\" href=\""
```

Comment:
Unsure what those styles were supposed to be, but those don't exist in this repo, nor in core. 404 is received as a response.

Reply:
thanks for cleaning up